pajbot/modules/__init__.py

```diff
@@ -66,7 +66,7 @@
 from pajbot.modules.slotmachine import SlotMachineModule
 from pajbot.modules.subalert import SubAlertModule
 from pajbot.modules.subscriber_fetch import SubscriberFetchModule
-from pajbot.modules.top import TopModule
+from pajbot.modules.topcommands import TopModule
```

Comment:
no renaming of modules - this will hcnage their id and nullify any settings of the module that a user already has. bad experience
can be fixed with a db migration but I won't ask you to do that, so just undo this change :)